src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,17 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
please see the message in the Slack channel and see the checklist.

Reply:
Sorry, done it